.travis.yml

```diff
@@ -2,5 +2,15 @@ language: node_js
 node_js:
   - 'stable'
   - 'lts/*'
-
-script: npm run lint && npm run test
+before_script:
+  - 'npm install grunt-cli -g'
+  - 'export DISPLAY=:99.0'
+  - 'sh -e /etc/init.d/xvfb start'
+  - 'sleep 3'
+script:
+  - 'npm run lint'
+  - 'npm run test'
+  - 'if [ ${TRAVIS_NODE_VERSION} = "stable" ]; then
```

Comment:
I assume cutting out into `runner.sh` or something is also better but I prefer to write this procedure here rather than that way. Because I'm not sure where I should put the file: I thought `tests/e2e/runner.sh` and `scripts/e2e-runner.sh` but both is not the best way because I thought this procedure is too short to cut out.